test/unit/instance_method_test.rb

```diff
@@ -98,7 +98,6 @@ def test_should_restore_original_method
     method.hide_original_method
     method.define_new_method
     method.remove_new_method
-    method.restore_original_method
```

Comment:
Since this test is named `test_should_restore_original_method` and it seems to have been testing the implementation in the parent class, `StubbedMethod`, presumably we could delete the whole test...?

Reply:
Same reasoning as https://github.com/freerange/mocha/pull/552/files#r978773035